homework-submission/Week1/App.js

```diff
@@ -0,0 +1,72 @@
+const ListItem = ({ description, deadline, className }) => {
+  return (
+    <li class={className}>
+      {description}, {deadline}
+    </li>
+  );
+};
+// The goal of this component is to create a single item of a list .
+
+const StaticList = () => {
+  return (
+    <div>
+      <ul>
+        {<ListItem description="Get out of bed" deadline="Wed Sep 13 2017" />}
+        {<ListItem description="Brush teeth" deadline="Thu Sep 14 2017" />}
+        {<ListItem description="Eat breakfast" deadline="Fri Sep 15 2017" />}
```

Comment:
I see there is no need for any of these braces here.

Braces in React are usually used to tell the jsx compiler this will be a javascript scope and therefore javascript involves to be processed.